purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1718,6 +1720,118 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
     }
 
     // endregion
+
+    // region Deprecated
+
+    /**
+     * The listener is responsible for handling changes to purchaser information.
+     * Make sure [removeUpdatedPurchaserInfoListener] is called when the listener needs to be destroyed.
+     */
+    @Deprecated(
+        "Renamed to updatedCustomerInfoListener",
+        level = DeprecationLevel.ERROR,
+        replaceWith = ReplaceWith("updatedCustomerInfoListener")
+    )
+    var updatedPurchaserInfoListener: UpdatedPurchaserInfoListener? = null
+        @JvmName("-deprecated_getUpdatedPurchaserInfoListener")
```

Comment:
great find! this makes me wanna get our API testers up for android, so we could easily confirm all of these

Reply:
Yes, because I have been testing them manually. Not sure we have the time before the release, but it's a good investment for the future to do these at some point in the near future